site.cfg.example

```diff
@@ -64,14 +64,14 @@
 #
 #   extra_compile_args
 #       Add additional arguments to the compilation of sources.
-#       Simple variable with no parsing done. 
+#       Parsed as if passed directly to a POSIX shell (even on Windows).
 #       Provide a single line with all complete flags.
 #           extra_compile_args = -g -ftree-vectorize
 #
 #   extra_link_args
 #       Add additional arguments when libraries/executables
 #       are linked.
-#       Simple variable with no parsing done. 
+#       Parsed as if passed directly to a POSIX shell (even on Windows).
```

Comment:
Is there a succinct way to describe what the difference is, or maybe give a small example?

Reply:
Difference from what prior state?

Reply:
please ignore